modules/data/pending_post.py

```diff
@@ -140,49 +140,68 @@ def get_votes(self, vote: bool) -> int:
+    def get_list_admin_votes(self, vote: 'bool | None' = None) -> 'Optional[list[str] | bool]':
         """Gets the list of admins that approved or rejected the post
 
         Args:
-            vote: whether you look for the approve or reject votes
+            vote: whether you look for the approve or reject votes, or None if you want all the votes
 
         Returns:
             list of admins that approved or rejected a pending post
         """
-        votes = DbManager.select_from(select="admin_id",
-                                      table_name="admin_votes",
-                                      where="g_message_id = %s and group_id = %s and is_upvote = %s",
-                                      where_args=(self.g_message_id, self.group_id, vote))
+
+        where = "g_message_id = %s and group_id = %s"
+        where_args = (self.g_message_id, self.group_id)
+
+        if vote is not None:
+            where += " and is_upvote = %s"
+            where_args = (self.g_message_id, self.group_id, vote)
+
+        votes = DbManager.select_from(select="admin_id, is_upvote",
+                                        table_name="admin_votes",
+                                        where=where,
+                                        where_args=where_args)
 
         if len(votes) == 0:  # the vote is not present
             return None
 
+        if vote is None:
+            return [(vote['admin_id'], vote['is_upvote']) for vote in votes]
+
         return [vote['admin_id'] for vote in votes]
 
-    def show_admins_votes(self, bot: Bot, approve: bool):
+
+    def show_admins_votes(self, bot: Bot):
         """After a post is been approved or rejected, shows the admins that approved or rejected it \
             and edit the message to delete the reply_markup
 
         Args:
             bot: bot
             approve: whether the vote is approve or reject
         """
-        admins = self.get_list_admin_votes(vote=approve)
-        text = "Approvato da:\n" if approve else "Rifiutato da:\n"
-        tag = '@' if Config.meme_get('tag') else ''
-        for admin in admins or []:
-            username = bot.get_chat(admin).username
-            text += f"{tag}{username}\n" if username else f"{bot.get_chat(admin).first_name}\n"
-
-        remaining_pending_posts = [
-            f"• https://t.me/c/{str(post.group_id)[4:]}/{post.g_message_id}\n"
-            for post in __class__.get_all(group_id=self.group_id)
-            if post.g_message_id != self.g_message_id
-        ]
-        text = f"Spot rimanenti:\n{''.join(remaining_pending_posts)}\n" if remaining_pending_posts else text
-
-        bot.edit_message_reply_markup(chat_id=self.group_id, message_id=self.g_message_id, reply_markup=None)
-        bot.send_message(chat_id=self.group_id, text=text, reply_to_message_id=self.g_message_id)
+
+        # fix circular import
+        get_post_outcome_kb = modules.utils.keyboard_util.get_post_outcome_kb
```

Comment:
I guess there is no real way around it.
To be honest, I feel like even the original method should not have been placed here. This is the _Data_ module, so it should only contain things strictly related to how the data is stored, but should not care about how the data is sent to the user.
I feel like this method could be moved in the `EventInfo` util (or some other ad-hoc util) and the pending post should be passed as a parameter.
This can also be done in a later PR.